src/symbols/Node.tsx

```diff
@@ -306,6 +306,7 @@ export const Node: FC<NodeProps> = ({
 
   const labelComponent = useMemo(
     () =>
+      labelVisible &&
```

Comment:
@ghsteff - can u confirm this change won't regress any of the other features we added?